### ISSUE TITLE
'zv' is inserted when the `c` command is followed by a search motion

### ISSUE THREAD

**Issue by nelstrom** [spiiph/vim-space#10]
When the `c` command is followed by a search motion, the characters `zv` are inserted.

Suppose the cursor was positioned on the opening '[' of the first line:

```
[submodule "vim/bundle/vim-space"]
    path = vim/bundle/vim-space
    url = git://github.com/spiiph/vim-space.git
```

Running: `c/v<CR>` produces the following:

```
zvvim/bundle/vim-space"]
    path = vim/bundle/vim-space
    url = git://github.com/spiiph/vim-space.git
```

With the cursor positioned after the 'zv'.

May be related to [Issue 1](https://github.com/spiiph/vim-space/issues#issue/1).

**Comment by nelstrom** [comment]
Just discovered that this bug can be avoided by including this line in .vimrc file:

```
let g:space_no_search = 1
```

Of course, that also means that `<Space>`/`<S-Space>` won't remap to n/N after using a search, but I can live with that for now.

**Comment by kongo2002** [comment]
You could try a commit I pushed into my fork of vim-space a few minutes ago. You can pull from here:
kongo2002/vim-space@9aab26c239f12e822e93

If it works for you maybe spiiph considers to pull my changes into upstream :)

**Comment by nelstrom** [comment]
I'll try that out, thanks!

**Comment by nelstrom** [comment]
I can confirm that [kongo2002's patch](https://github.com/kongo2002/vim-space/commit/9aab26c239f12e822e93) fixes this issue for me. I'll continue using that fork, and report here if it introduces any other issues.

Thanks kongo2002!

**Comment by spiiph** [comment]
Clone of issue #1. Fixed by @kongo2002 in 9aab26c2.
